saas/{{cookiecutter.project_name}}/webapp/src/lib/utils/strings.ts

```diff
@@ -0,0 +1,3 @@
+export function capitalizeFirstLetter(text: string) {
+    return text.charAt(0).toUpperCase() + text.slice(1);
+}
```

Comment:
This is the classic one liner place with arrow functions... I guess works best also for reading something like:
```js
export const capitalizeFirstLetter = (text:string) => text.charAt(0).toUpperCase() + text.slice(1)
```